js/apps/system/_admin/aardvark/APP/react/src/components/form/InputControl.tsx

```diff
@@ -0,0 +1,19 @@
+import { BaseFormControlProps, FormikFormControl } from "./FormikFormControl";
+import { useField, useFormikContext } from "formik";
+import React from "react";
+import { Input, InputProps } from "@chakra-ui/react";
+
+export type InputControlProps = BaseFormControlProps & {
+  inputProps?: InputProps;
+};
+
+export const InputControl = (props: InputControlProps) => {
```

Comment:
Same as above, I don't think it matters if it's default export or not. Sometimes named exports might be better. It sounds like a preference at this point, I wouldn't want to make change based on that

Reply:
It's not just my preference, but commonly used convention. Many well known libraries, including React and Lodash provide default exports where it makes sense (for example - in cases where the module name reflects its primary function).

Reply:
While these libraries do offer multiple different ways of importing, many of them exist for legacy reasons.

- A pitfall of using default exports is that it can be named anything in usage
- An advantage of using a named export is that it can allow IDEs to quickly suggest autocomplete

Common convention is not always the best, because the reasons behind them could be related to legacy code; not wanting breaking changes etc